app/models/user.rb

```diff
@@ -1,8 +1,12 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable
+
```

Comment:
- [ ] Please make sure the devise is also confirmable and ask for confirmation of email as requested in the project requirements.

![image](https://github.com/diarisdiakite/Blog-app/assets/83803414/abe05b4c-d8fb-45b0-9afc-b0bee093d7b9)